tests/staticfiles_tests/test_management.py

```diff
@@ -122,6 +124,14 @@ def test_local_storage_detection_helper(self):
             storage.staticfiles_storage = staticfiles_storage
 
 
+class TestRunserver(StaticFilesTestCase):
```

Comment:
It's not obvious to me why you put the class here -- it looks like it's in the middle of two collectstatic test classes, perhaps it could be better ordered.